packages/test/test-end-to-end-tests/src/test/containerRuntime.spec.ts

```diff
@@ -120,6 +120,7 @@ describeCompat(
 						minimumBatchSizeInBytes: compression ? 1000 : Infinity,
 						compressionAlgorithm: CompressionAlgorithms.lz4,
 					},
+					enableGroupedBatching: compression, // Compression w/o grouping is not supported
```

Comment:
@MarioJGMsoft I think we need to revisit this too - I think there's an existing gap in test coverage where "Grouping Disabled" isn't covered

Reply:
I think this coverage is better placed in compression.spec.ts actually.  See AB#6287. those tests are disabled for cross-version compat, but we can reenable them with some work.